src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends AbstractFigure {
```

Comment:
Don't begin class or method implementation with an empty line. Remove the empty line at the beginning of the class definition (line 2).